src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements HasDiagonal, Drawing {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void infoAboutFigure() {
```

Comment:
what is the difference between draw() method and getInfo() method?
these methods don't save any state. They only create the string using data from class and return it. - it's a behavior
We can use the method getInfo() with figures or people or house or smth else. It doesn't matter.

Reply:
done